### ISSUE TITLE
Agenda cells should conform to Apple's accessibility guidelines

### ISSUE THREAD

**Issue by mdiasdev** [SwiftFest/ios-app#65]
In the Agenda table view cells, the speaker’s image should be moved above the text in the cell or hidden altogether when one of the accessibility fonts is used. This is not a “must”, but follows Apple’s recommendations to limit displayed content when larger fonts are used. The image eats up a disproportionate amount of horizontal space.

**Comment by ZevEisenberg** [comment]
I heard a great shortcut for this recently: put the text and the image in a horizontal stack view, and switch the axis to vertical if the current preferred content size is one of the “accessibility” extra-large sizes.
